Base.rte/AI/HumanAI.lua

```diff
@@ -6,13 +6,17 @@ function Create(self)
 	self.AI = NativeHumanAI:Create(self);
 	--You can turn features on and off here
 	self.armSway = true;
-	self.automaticEquip = true;
```

Comment:
I initially made it because I was playing with a friend who didn't realize in the midst of battle that their weapons were getting destroyed, so it's more of a noob aid... tbh, it could go as a gameplay setting, whaddya think?

Reply:
Oh, it applies to player controlled actors too, eh? I definitely don't think it should be on by default, but it could be a player specific setting I guess, since adding that doesn't affect people who don't want it. If we do go that route, it'd probably still be better in cpp though, since it'd be built into settings menus or whatever.